xla/service/gpu/ir_emission_utils.cc

```diff
@@ -136,6 +141,22 @@ bool IsMatrixMultiplication(const HloInstruction& dot) {
   return true;
 }
 
+int64_t MinThreadsXRowReduction(const HloModuleConfig& hlo_module_config) {
+#ifdef GOOGLE_CUDA
+  auto ptxas_config =
+      PtxOptsFromDebugOptions(hlo_module_config.debug_options());
+  auto ptxas_version_tuple =
+      se::GetAsmCompilerVersion(ptxas_config.preferred_cuda_dir);
+  // ptxas versions prior to 12.2 have a bug when register spilling occurs, so
```

Comment:
`ptxas versions prior to 12.2 have a bug when register spilling occurs` ->
`ptxas versions prior to 12.2 have a very rare bug when very high register spilling occurs with some order of instructions,`

Reply:
Thanks @nouiz!